dns-controller/cmd/dns-controller/main.go

```diff
@@ -73,6 +74,8 @@ func main() {
 	flag.StringVar(&gossipListenSecondary, "gossip-listen-secondary", fmt.Sprintf("0.0.0.0:%d", wellknownports.DNSControllerGossipMemberlist), "address:port on which to bind for gossip")
 	flags.StringVar(&gossipSecretSecondary, "gossip-secret-secondary", gossipSecret, "Secret to use to secure gossip")
 	flags.StringSliceVar(&gossipSeedsSecondary, "gossip-seed-secondary", gossipSeedsSecondary, "If set, will enable gossip zones and seed using the provided addresses")
+	flags.BoolVar(&ipv4, "ipv4", ipv4, "Internal network has IPv4")
+	flags.BoolVar(&ipv6, "ipv6", ipv4, "Internal network has IPv6")
```

Comment:
Maybe good to be more specific?
```suggestion
	flags.BoolVar(&ipv4, "internal-ipv4", internal-ipv4, "Internal network has IPv4")
	flags.BoolVar(&ipv6, "internal-ipv6", internal-ipv6, "Internal network has IPv6")
```